editor/themes/editor_theme_manager.cpp

```diff
@@ -1451,8 +1451,44 @@ void EditorThemeManager::_populate_standard_styles(const Ref<EditorTheme> &p_the
 	}
 
 	// SpinBox.
-	p_theme->set_icon("updown", "SpinBox", p_theme->get_icon(SNAME("GuiSpinboxUpdown"), EditorStringName(EditorIcons)));
-	p_theme->set_icon("updown_disabled", "SpinBox", p_theme->get_icon(SNAME("GuiSpinboxUpdownDisabled"), EditorStringName(EditorIcons)));
+	{
+		Ref<Texture2D> empty_icon = memnew(ImageTexture);
+		p_theme->set_icon("updown", "SpinBox", empty_icon);
```

Comment:
Is setting empty icon needed for anything? At worst it will be null, but that can be (and should be) handled in code.

Reply:
So far, I noticed that the generated empty icon was used elsewhere in the theme. If I don't do this, the value is inherited from the parent theme. I noticed the technique is used elsewhere, so I took my inspiration from it. Nothing new about generating empty icons in the editor theme.
For this specific case, I want to avoid breaking existing themes that rely on the combined up/down icon, while setting it to empty by default.

Reply:
If I don't set anything, it inherits from the parent theme instead. In this specific case, I want to avoid breaking the existing projects that may still have customized the combined up/down icon. In the updated theme, as there are the individual arrows now available, I want to get rid of the combined up/down.
There should be a way to say "nothing", distinct from the default "inherit" behavior", without having to create an empty resource just for that, I agree.

This empty icon technique is not invented by me; I spotted it elsewhere in the theme and copied it over.